scripts/create-container.sh

```diff
@@ -0,0 +1,137 @@
+#!/bin/bash
+# Createded date: 21/03/2016
+
+# Flags
+# set -e
+
+# Default values
+name="ofn-test"
+template="/usr/share/lxc/templates/lxc-ubuntu"
+rls="trusty"
+dconfig="/tmp/ubuntu.$name.conf"
+host="ofn-test.org"
+nproject="openfoodnetwork"
+fproject="${PWD%/*}/$nproject"
+app_user="openfoodnetwork"
```

Comment:
Do we need these variables defined here? We'll get them with the next `source` right?